examples/express/index.ts

```diff
@@ -0,0 +1,135 @@
+import express, { Router } from 'express'
+import SgidClient from '@opengovsg/sgid-client'
+import * as dotenv from 'dotenv'
+import crypto from 'crypto'
+import cookieParser from 'cookie-parser'
+import { fetchStaticFiles } from './helpers'
+import open from 'open'
+
+dotenv.config()
+
+const PORT = 5000
+
+const sgid = new SgidClient({
+  clientId: String(process.env.SGID_CLIENT_ID),
+  clientSecret: String(process.env.SGID_CLIENT_SECRET),
+  privateKey: String(process.env.SGID_PRIVATE_KEY),
+  redirectUri: `http://localhost:${PORT}/api/callback`,
+})
+
+const app = express()
+
+const apiRouter = Router()
+
+const SESSION_COOKIE_NAME = 'exampleAppSession'
+const SESSION_COOKIE_OPTIONS = {
+  httpOnly: true,
+}
+
+type SessionData = Record<
+  string,
+  | {
+      nonce?: string
+      // Store state as search params to easily stringify key-value pairs
+      state: URLSearchParams
+      accessToken?: string
+    }
+  | undefined
+>
+
+/**
+ * In-memory store for session data.
+ * In a real application, this would be a database.
+ */
+const sessionData: SessionData = {}
```

Comment:
nit: have we considered using a `Map` instead of an object? in a demo app like this it doesn't really matter but using a `Map`:
1. is more suited for a use case like this than object (due to [performance](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map))
2. we don't need to explictly type the Record value as undefined
```suggestion
type Session = {
      nonce?: string
      state: URLSearchParams
      accessToken?: string
}

const sessionData = new Map<string, Session>()
```

however, `Map` is definitely not as commonly used as objects so i'm afraid that making this change might make it less understandable for some readers